integration-tests/tests/resource-observer/resource-obs.spec.js

```diff
@@ -136,5 +136,52 @@ module.exports = {
     const afterLoadImage = imageSpans.find( span => span.tags['component'] === 'splunk-post-doc-load-resource');
 
     await browser.assert.notEqual(docLoadImage.traceId, afterLoadImage.traceId);
+  },
+  'should propagate tracing context to created spans': async function(browser) {
```

Comment:
i see that this suite is currently disabled, but i enabled it locally and everything passed